Readme.md

```diff
@@ -28,7 +31,7 @@ Why focus on soldering when you can play with the settings instead? With this ap
 
 - python 3 (tested with 3.10)
 - pipenv
-- a browser
+- a browser that supports [BLE GATT](https://caniuse.com/mdn-api_bluetooth)
```

Comment:
@River-b the browser doesn't need BLE - the bluetooth is handled by the backend entirely